File: seongwon/LV1/부족한금액계산하기.js
function solution(price, money, count) {
  let total = 0;
  for (let i = 1; i <= count; i++) {
    total = total + price * i;
  }
  return total > money ? total - money : 0;
}
console.log(solution(3, 20, 4));

File: jisoo/programmers/level1/부족한_금액_계산하기.py
"""
https://school.programmers.co.kr/learn/courses/30/lessons/82612
"""


def solution(price, money, count):
  answer = -1

  sum = 0
  for i in range(count):
    sum += (i + 1) * price

  print(sum)
  return max(0, sum - money)


#다른 풀이
def solution2(price, money, count):
  return max(0,price*(count+1)*count//2-money)
